ray_on_golem/server/services/golem/helpers/demand_config.py

```diff
@@ -82,9 +83,10 @@ async def _get_image_url_and_hash(self, demand_config: DemandConfigData) -> Tupl
             return image_url, image_hash
 
         if image_tag is None:
+            rog_version = pkg_resources.get_distribution("ray-on-golem").version
             python_version = platform.python_version()
```

Comment:
The python version will be the one we use for the image building?
If not, maybe we could force `image_tag` to be present?
Or maybe either of `image_tag`, `image_url` or `image_hash`? (while keeping proper `image_tag` in the minimal yaml
(unless we can work out the default in a way it will match the python we used for image creation (I guess it is the one that should be put in the tag)

Reply:
I planned to provide `image_tag` in all yamls as this feature of auto-generating tags faces a lot of issues in real use cases. This change just updated it to the newest format.

Reply:
right, sorry for jumping between vodka and an appetizer :rofl: I am backing off now :pray: